vipps-recurring-api.md

```diff
@@ -1542,7 +1645,8 @@ This API returns the following HTTP statuses in the responses:
 | HTTP status                | Description                                                       |
 |----------------------------|-------------------------------------------------------------------|
 | `200 OK`                   | Request successful                                                |
-| `201 Created`              | Request successful, resource created                              |
+| `204 No content`           | Request successful                                                |
+| `202 Accepted`             | Request accepted, the requested action will be attempted          |
```

Comment:
```suggestion
| `202 Accepted`             | Request accepted, the action will likely succeed but has not yet been enacted          |
```

Reply:
202 Accepted  status code indicates that the request has been accepted for processing, but the processing has not been completed.